README.md

```diff
@@ -10,18 +10,45 @@ A virtual machine is used to encapsulate Docker dependencies. `docker-compose` i
 - VirtualBox 5.0+
 - Ansible 1.8+ (on host)
 - AWS CLI 1.10+
+- AWS Account (to store artifacts, secrets)
+- [Auth0 Account](https://auth0.com/) (user management)
+- [Rollbar Account](https://rollbar.com/) (error reporting -- optional)
 
-On your host machine you need to set up a `raster-foundry` profile for the Raster Foundry AWS account using the following command:
+#### Setting Up AWS Account
 
+There are a set of tasks necessary before starting development in order to provision Raster Foundry. Raster Foundry depends heavily on AWS resources and using AWS resources to manage secrets/containers/artifacts in development. If only local development is being done, the primary resource that will be used are S3 buckets to store secrets.
+
+In the AWS account you need to create a few buckets for the following:
+ - A bucket to house raw data (e.g. geotiffs, JPEG2000, ingest definitions, etc.)
+ - A config bucket that will store secrets for development and or other environments, an exported database for development data
+ - A bucket to house processed data (e.g. thumbnails, processed raster RDDs)
+
+The names of the buckets are not important, but they should be memorable and easy to parse for your own sake. On your host machine you need to set up an AWS profile for the account with the S3 buckets. For instance, to set up an AWS profile called `raster-foundry` with the AWS cli the following command would be used:
 ```
 $ aws configure --profile raster-foundry
 ```
 
 You will be prompted for an access key and secret key.
 
+#### Setting Development Environment Variables
+
+The `.env.template` file is a template file with environemnt variables that get injected into running containers during development. This file should be copied into the AWS config bucket created after filling in sensitive information (replacing all `PLACEHOLDER` values with appropriate values for your AWS setup). When provisioning this file is copied to the development environment and injected into containers with `docker-compose`.
+
+In addition to setting up an AWS account, you must register for an Auth0 account to produce secrets to use in the `.env` file. You need to go through setting up an application and copying over the client IDs, domain, and secret. Additionally, you need to generate a management API app to handle managing the generation of [refresh tokens for users via the management API](https://auth0.com/docs/api/management/v2/tokens).
```

Comment:
Good point, fixed in https://github.com/azavea/raster-foundry/pull/1269/commits/3dcf90a87538b444ac6fd9a9e0809ffb916e1d69